Std/Data/Rat/Lemmas.lean

```diff
@@ -11,6 +11,11 @@ import Std.Tactic.SeqFocus
 
 namespace Rat
 
+attribute [local ext] Rat
+
+@[simp] theorem mk_den_one {r : Int} :
```

Comment:
Excellent simp lemma! The name is still unusual but there are probably no uses outside simp, so it doesn't matter much.